src/test/java/nextstep/subway/acceptance/LineAcceptanceTest.java

```diff
@@ -100,111 +55,119 @@ void getLines() {
      * When 생성한 지하철 노선 조회를 요청 하면
      * Then 생성한 지하철 노선을 응답받는다
      */
-    @DisplayName("지하철 노선 조회")
     @Test
-    void getLine() {
+    void 지하철_노선_조회() {
         // given
-        Map<String, String> params = new HashMap<>();
-        params.put("name", "9호선");
-        params.put("color", "갈색");
-
-        ExtractableResponse<Response> createResponse =
-                RestAssured.given().log().all()
-                        .body(params)
-                        .contentType(MediaType.APPLICATION_JSON_VALUE)
-                        .when()
-                        .post("/lines")
-                        .then()
-                        .log().all()
-                        .extract();
+        ExtractableResponse<Response> 지하철_노선_생성_응답 = LineSteps.지하철_노선_생성_요청("9호선", "갈색");
 
         // when
-        String url = createResponse.header("Location");
-        ExtractableResponse<Response> response =
-                RestAssured.given().log().all()
-                        .accept(MediaType.APPLICATION_JSON_VALUE)
-                        .when()
-                        .get(url)
-                        .then().log().all()
-                        .extract();
-
-        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
-        String lineName = response.jsonPath().getString("name");
+        String lineId = getLineId(지하철_노선_생성_응답.header("Location"));
```

Comment:
이왕 메서드로 분리한거 더 할 수 있지 않을까요?
```suggestion
        String lineId = getLineId(지하철_노선_생성_응답);
```